### PR TITLE
Enable KET to run in parallel, finally fix ssh default

### PR DESCRIPTION
**Pull request by dkoshkin**
* Fixes a few things that would prevent KET from running in parallel. 
  * Don't generate run files in the `ansible` directory
  * Prefix the runs directories with the cluster name
* Finally fixes the `ssh.key` not having to be an absolute path, it now defaults to the key inside the `$HOME/.ssh` directory.

### CODE REVIEW

**Comment by emmetthitz** [comment]
So who's writing the tests for this?

**Comment by dkoshkin** [comment]
@alexbrand made some changes PTAL

File: pkg/install/validate.go [hunk 1]
```diff
@@ -217,9 +217,6 @@ func (s *SSHConfig) validate() (bool, []error) {
 	if _, err := os.Stat(s.Key); os.IsNotExist(err) {
 		v.addError(fmt.Errorf("SSH Key file was not found at %q", s.Key))
 	}
-	if !filepath.IsAbs(s.Key) {
-		v.addError(errors.New("SSH Key field must be an absolute path"))
```
[the remark this answers is not in the document]
Comment (dkoshkin) [reply pkg/install/validate.go]: Added a new test, thanks for pointing that out

**Comment by alexbrand** [comment]
LGTM

File: pkg/install/validate.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmetthitz) [reply pkg/install/validate.go]: This was actually about to be an issue for the provisioner as well. Cool beans. 

The only concern I have about this is making sure we remove keys that are no longer in use appropriately so the cluster lifecycle stuff doesn't end up needing off-site key rotations or something equally ugly.

**Pull request merged**